src/clj/rems/api/public.clj

```diff
@@ -61,4 +61,6 @@
       (ok (assoc env
                  :authentication "HIDDEN"
                  :database-url "HIDDEN"
-                 :test-database-url "HIDDEN")))))
+                 :test-database-url "HIDDEN"
+                 :oidc-client-id "HIDDEN"
```

Comment:
The client ID is public information, so it doesn't need to be hidden. It's present in the login redirect.

Reply:
right! fixed